### ISSUE TITLE
hostname 'sec.ipty.de' doesn't match either of 'project.ipty.de', 'ipty.de'

### ISSUE THREAD

**Issue by xh3b4sd** [giantswarm/german-holiday#5]
Just found this issue.

Requesting the python server from within the container. 

```
# curl 0.0.0.0:5000
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request.  Either the server is overloaded or there is an error in the application.</p>
```

Produced log of the failed request

```
an error occured: hostname 'sec.ipty.de' doesn't match either of 'project.ipty.de', 'ipty.de'
```

**Comment by zeisss** [comment]
still the case.

**Comment by zeisss** [comment]
Probably http://docs.python-requests.org/en/latest/community/faq/#what-are-hostname-doesn-t-match-errors

> Python3 and Python 2.7.9+ include native support for SNI in their SSL modules. For information on using SNI with Requests on Python < 2.7.9 refer to this Stack Overflow answer.

**Comment by zeisss** [comment]
Fixed with 4f88831 - will deploy then close

**Comment by zeisss** [comment]
Up and running. http://istodayaholidayingermany.com/
